bin/image-create-openstack.sh

```diff
@@ -1,135 +1,51 @@
 #!/usr/bin/env bash
 
 # Checks and uploads specified Image to user's OpenStack account.
-# Downloads image first from an Amazon S3 storage account to host,
-# then uploads image to openstack teenancy.
-# Uses curl and python-glanceclient to do the job.
+#
+# 1. Downloads image first from an Amazon S3 storage (using curl and python-glance client)
+#
+# 2. Converts the image to the required format (if needed)
+#
+# 3. Uploads image to openstack tenancy. (using curl and python-glance client)
+#
+#
 #
 # Env vars
+#
 #   KN_IMAGE_NAME
+#   KN_DISC_FORMAT
```

Comment:
To avoid confusion with image format on image downloaded change "disk_format" to "glance_disk_format"